primitives/types/src/lib.rs

```diff
@@ -27,7 +27,7 @@ use sp_runtime::{
 	traits::BlakeTwo256,
 	OpaqueExtrinsic,
 };
-use std::vec::Vec;
+use sp_std::vec::Vec;
```

Comment:
With this change, is the `sgx_tstd` necessary any more in the types primitives? If not, I think it makes sense to remove it completely.  @mullefel or is there any reason to keep the sgx feature inside the types crate? (no `this_error` in the types crate 😋 )

Reply:
(check out https://github.com/integritee-network/worker/issues/841)